docs/api/cloudrootfs.md

```diff
@@ -13,44 +13,28 @@ cloud rootfs will package all the dependencies refers to the kubernetes cluster
 │   ├── kubectl
 │   ├── kubelet
 │   ├── nerdctl
+│   ├── kubelet-pre-start.sh
+│   ├── helm
```

Comment:
What if a dual-architecture cluster image, such as support for both amd64 and arm64?

I think we should have a relatively complete example.